torchci/pages/benchmark/llms.tsx

```diff
@@ -320,7 +340,7 @@ export default function Page() {
     <div>
       <Stack direction="row" spacing={2} sx={{ mb: 2 }}>
         <Typography fontSize={"2rem"} fontWeight={"bold"}>
-          LLMs Benchmark DashBoard
+          Benchmark DashBoard
```

Comment:
@huydhn nit: To be consistent with TorchInduction, TorchAO, can we rename this name to inlcude "ExecuTorch", i.e. "ExecuTorch Performance DashBoard"?

Reply:
Yeah, let me follow up with a PR to add better title to these dashboards